Lib/asyncio/constants.py

```diff
@@ -11,3 +13,10 @@
 
 # Number of seconds to wait for SSL handshake to complete
 SSL_HANDSHAKE_TIMEOUT = 10.0
+
+# The enum should be here to break circular dependencies between
+# base_events and sslproto
+class _SendfileMode(enum.Enum):
```

Comment:
This feels like a wrong place to define an enum.  Maybe base_events?

Reply:
See comment above about circular deps.
sslproto requires the enum, base_events uses it too.

Moreover they do import each other.
It's a bad smell but I don't want to fix imports in this PR

Reply:
OK.